### PR TITLE
Move help to zanata.org, updates on footer

### PR DESCRIPTION
**Pull request by aeng**
https://bugzilla.redhat.com/show_bug.cgi?id=1195384

### CODE REVIEW

File: zanata-war/src/main/java/org/zanata/action/ApplicationConfigurationAction.java [hunk 1]
```diff
@@ -57,4 +58,18 @@ public boolean isSingleOpenId() {
...
     }
+
+    public String getHelpUrl() {
+        if(!StringUtils.isEmpty(applicationConfiguration.getHelpUrl())) {
+            return applicationConfiguration.getHelpUrl();
+        }
+        return ServerConfigurationBean.DEFAULT_HELP_URL;
+    }
+
+    public String getTermUrl() {
```
Comment (carlosmunoz) [inline zanata-war/src/main/java/org/zanata/action/ApplicationConfigurationAction.java]: Lets call the methods the same: `getTermsOfUseUrl`

File: zanata-war/src/main/webapp/WEB-INF/template/template.xhtml [hunk 2]
```diff
@@ -105,51 +105,7 @@
...
-                  applicationConfiguration.scmDescribe)}
-                  | #{msgs['jsf.CopyrightNotice']}"
-                  escape="false"/>
-              </p>
-            </div>
-          </div>
-        </div>
-      </div>
-    </footer>
+    <ui:include src="footer.xhtml"/>
```
Comment (carlosmunoz) [inline zanata-war/src/main/webapp/WEB-INF/template/template.xhtml]: Does this mean we were duplicating the same code in `footer.xhtml` and this page?

Comment (aeng) [inline zanata-war/src/main/webapp/WEB-INF/template/template.xhtml]: Yes. We got 2 different footer (old and new template)

File: zanata-war/src/main/resources/messages.properties [hunk 3]
```diff
@@ -720,8 +720,7 @@ jsf.RemovedAPermission={0} has been removed as {1}
 
 #------ [home] > Help ------
 #------ [home] > Help > Edit Help Page Content ------
-jsf.EditHelpPageContent=Edit Help Page Content
-jsf.ContactAdmin=Contact Admin
+jsf.ContactAdmin=Contact admin
```
Comment (djansen-redhat) [inline zanata-war/src/main/resources/messages.properties]: Why did this go to lower case?

Comment (aeng) [inline zanata-war/src/main/resources/messages.properties]: to be consistence with other text in footer.

**Comment by djansen-redhat** [comment]
:white_check_mark: Tested

**Comment by aeng** [comment]
Merged into integration/master. https://github.com/zanata/zanata-server/commit/e6d4bd712ba7a5ea49ec25842f8cf86836f41637

**Pull request closed without merging**